en/config_heli/README.md

```diff
@@ -0,0 +1,87 @@
+# Helicopter Configuration
+
+This section contains topics related to helicopter configuration and tuning.
+
+:::note
+Helicopters are less well-supported than other types of vehicles.
+We would welcome your [contribution](../contribute/README.md) of new features, new frame configurations, or other improvements.
+:::
+
+## Supported Configurations
+
+Supported helicopter configurations:
+- Single main rotor with any type of swash-plate
+- ESC tail rotor
+
+
+## Setup
+
+To setup and configure a helicopter: 
+
+1. Select the helicopter [Airframe](../config/airframe.md) in QGroundControl
+1. Configure [Actuator](../config/actuators.md) geometry
+   ![Geometry helicopter](../../assets/config/actuators/qgc_geometry_helicopter.png)
```

Comment:
Would you happen to have a complete configuration screen for the helicopter? ("as well")?

Reply:
@bkueng If you have a setup screen for the whole helicopter actuator config that would be good (i.e. for a real build). I would use it for a separate helicopter config page, but below the current actuator config. In a further PR.